### DISCUSSION TITLE
[Feature Request] Lay command

### DISCUSSION THREAD

**Discussion by Malachiel87** [Zrips/CMI#960] [Feature Requests]
An addition to sit command, would be like /lay command, it simulate a fake bed. amd  you lay yourself on floor, it would be possible? There is already a plugin that do it but is abandoned and lay work only for other players but you will see not layed.

**Comment by Baktus79** [comment]
That would be a great addition to the /sit command :D

**Comment by DragonisOmega** [comment]
/lay back and /lay stomach or /lay, /crawl

**Reply by AZFire1** [reply]
This is still wanted. Add this pls :)

**Comment by Zrips** [comment]
Not as simple feature as it might look like from first glance.

**Reply by AZFire1** [reply]
Is there a chance of it being added though? Or, should I look for another plugin to do this for me?

**Reply by Zrips** [reply]
For time being, just look for another plugin until this gets implemented

**Comment by Recliness** [comment]
this would be amazing!

**Comment by KanakxOP** [comment]
still hoping for this feature
